src/main/resources/META-INF/plugin.xml

```diff
@@ -3,7 +3,25 @@
     <name>Coding Assistant</name>
```

Comment:
Please, rename it please into _Coding Assistant Algorithm_ or better _Hint Generation Algorithm_. The motivation of it is creating a separated plugin only for UI